src/App.js

```diff
@@ -1,5 +1,81 @@
+import { Console } from "@woowacourse/mission-utils";
+
 class App {
-  async run() {}
+  async run() {
+    try {
+      const input = await this.readInput();
+      const result = this.calculate(input);
+      Console.print(`결과 : ${result}`);
+    } catch (error) {
+      Console.print(error.message);
+      throw error;
+    }
+  }
+
+  async readInput() {
+    return await Console.readLineAsync("덧셈할 문자열을 입력해 주세요:\n");
+  }
+
+  calculate(input) {
+    // 1번 기능: 빈 문자열일 경우 0을 반환
+    if (input === "") {
+      return 0;
+    }
+
+    let numbers = [];
+    let delimiterPattern = /[,|:]/; // 기본 구분자 쉼표(,) 또는 콜론(:)
```

Comment:
```suggestion
    let delimiterPattern = /[,:]/; // 기본 구분자 쉼표(,) 또는 콜론(:)
```
`|` 기호 없어도 `/[,:]/`만으로 표현해도 처리할 수 있습니다.